app/views/content/is-teaching-right-for-me/behaviour-management.md

```diff
@@ -0,0 +1,95 @@
+
+## How can I create a positive learning environment?
+In my experience of teaching hundreds of thousands of children, if they’re engrossed in their learning and know what it is they need to be doing, they'll be much less disruptive. The biggest challenge isn't usually big things it’s more the chatting and low-level disruption.  
+
+Make sure you’re very clear. If children understand your expectations of them in terms of workload, behaviour and outcome and you scaffold that journey then behaviour becomes much less of a problem. Start off by modelling what you would want in your classroom. Do you want the children lining up at the start of the lesson? Or do you want them coming straight in and starting with the work?
+
+Have routines and rituals or standard operational procedures so children know when they go to your class, this is what they do. So they understand what's required from them and know they'll be rewarded if they get that behaviour right.
+
+## How do I manage more disruptive behaviour?
+Managing behaviour is a learning curve, and exactly what a teacher training course is designed for. You'll be offered practical guidance and support so don’t worry if you don’t get it right every time.
+
+It’s all about the relationships you build with pupils. If they know they’ll get away with disruptive learning, they'll disrupt learning.
+You have to set and communicate firm but fair boundaries so they know they're safe and won't be ridiculed or embarrassed and their learning comes first. They’ll push against these boundaries because that’s what children do. But if they find they meet with a brick wall they won't push any further because they know what's going to happen in any lesson they turn up to. 
+
+And follow up on infringements, regardless of how long it takes to reach the result you intend. But don’t threaten anything you're not able to follow up with, like detention for the rest of their life!
+
+## How do I build relationships with pupils?
+Often building relationships with pupils happens outside of the classroom. It’s having conversations at break time in the corridor or as the kids are coming into school or at after school club. And talking to pupils, not just about what they're learning from you, but taking an interest in them as people. What did they have for their tea last night? How’s their poorly dog? What kind of things that make them laugh or sad and getting to know their authentic self.
+
+But it has to be a two-way thing, you’ve got to give a little of yourself so they can empathise. So you could share your hopes and dreams. And if you've got a cat at home, that always helps.
+
+## How can I reward good behaviour?
+Children tend to respond well to praise. It might be telling them they've done really well, or giving them a merit or a community point or whatever your school does. It's this kind of positive reinforcement that you would constantly do.
```

Comment:
```suggestion
Children tend to respond well to praise. It might be telling them they've done really well, or giving them a merit or a community point or whatever your school does. You would constantly do this kind of positive reinforcement.
```
Last sentence feels a bit awkward - is it worth rejiging?